### PR TITLE
Add note on resolving host names

### PR DESCRIPTION
**Pull request by rhomast**

### CODE REVIEW

**Comment by rhomast** [comment]
@kminder Does it matter if you do this before or after you install Verrazzano?

File: content/en/docs/setup/private-registry/private-registry.md [hunk 1]
```diff
@@ -161,6 +161,12 @@ You must have the following software installed:
 
   Use the Verrazzano CLI to install Verrazzano from your private registry.
 
+{{< alert title="NOTE" color="primary" >}}
+To ensure that the host names used for the system console are resolvable, you must configure one of the following:
```
Comment (kminder) [inline content/en/docs/setup/private-registry/private-registry.md:165]: consoles?

File: content/en/docs/setup/private-registry/private-registry.md [hunk 2]
```diff
@@ -161,6 +161,12 @@ You must have the following software installed:
 
   Use the Verrazzano CLI to install Verrazzano from your private registry.
 
+{{< alert title="NOTE" color="primary" >}}
```
Comment (kminder) [inline content/en/docs/setup/private-registry/private-registry.md:164]: I think this alert is fine where it is.  However it probably needs some additional context to be more helpful.  Unfortunately I don't have a good suggestion for what other doc should be linked.
Some of the information about what host names need to be resolvable is here
https://verrazzano.io/devel/docs/setup/access/console-urls/
Some of the information about customizing DNS is here
https://verrazzano.io/devel/docs/networking/traffic/dns/

Comment (rhomast) [reply content/en/docs/setup/private-registry/private-registry.md:164]: I've updated the note to add links to both.

**Pull request merged**